### DISCUSSION TITLE
Scrollable Frame only if content is too big

### DISCUSSION THREAD

**Discussion by aditydcp** [TomSchimansky/CustomTkinter#2293] [Q&A] [answered]
Hi, I just wanted to ask, is it possible to have a Scrollable Frame only when the content is too big?
So if the content isn't too big, it looks just like a simple frame, but if the content gets too big, it will start to show a scrollbar. Just like how the CTk.Textbox behaves

**Comment by Akascape** [comment, marked as answer]
@aditydcp Try CTkXYFrame widget: https://github.com/Akascape/CTkXYFrame

the scrollbar hides automatically and both axis are there.

**Reply by aditydcp** [reply]
That's neat! Thank you.

However I'm having trouble with having the content inside it expand/fill the available space.
With the usual CTkFrame, I can just use `fill` argument on its content to make it fill the available space.
Here's an example of what I want to achieve:
```
root = customtkinter.CTk()
root.geometry("400x400")

xy_frame = CTkXYFrame(root, border_color="blue", border_width=5)
customtkinter.CTkLabel(xy_frame, text="Title", bg_color="red").pack(fill="x")
xy_frame.pack(fill="both", expand=True)

usual_frame = customtkinter.CTkFrame(root, border_color="green", border_width=5)
customtkinter.CTkLabel(usual_frame, text="Title", bg_color="red").pack(fill="x")
usual_frame.pack(fill="both", expand=True)

root.mainloop()
```
![ss](https://github.com/TomSchimansky/CustomTkinter/assets/57148744/f68495e6-e7f1-4069-880a-e67fad91ca29)

As seen right here, I wanted to get the content to fill the space taken by the frame (like the one on bottom). Is there a way to achieve this with your widget?
Thank you in advance!
